### ISSUE TITLE
Does this take lots of resources?

### ISSUE THREAD

**Issue by anon238** [krehwell/HeckarNews#6]
What kind of plan should I think of? 512 MB RAM, 25 GB storage etc? Nice job by the way. Yours looks better than the most.

**Comment by krehwell** [comment]
hmm, not sure about how to answer this question. But I think it shouldn't take a lot of storage since most of data is stored on atlas (mongoDB cluster cloud). 

about 512MB RAM, the only thing to consider here is node usually takes up much ram than expected, so if you could have more than that it should ad least assure that everything can run smoothly

**Comment by anon238** [comment]
Should this be enough? 

2GB Memory / 60GB Storage / 3TB Bandwidth €18 a month or is that too much and expensive for this project?

**Comment by krehwell** [comment]
yeah, I think it's enough though, and probably too much too for the storage. Not sure about the bandwidth, I just host it on vercel and let vercel take care of the rest

**Comment by ghost** [comment]
> Should this be enough?
> 
> 2GB Memory / 60GB Storage / 3TB Bandwidth €18 a month or is that too much and expensive for this project?

I deployed, getServerSideProps not support Static HTML Export (https://nextjs.org/docs/advanced-features/static-html-export#unsupported-features), So much larger cpu and memory footprint. I have done a stress test, and I feel that the configuration of vercel is not enough, and sometimes it stops responding. I think you can decide the configuration according to the number of visitors and the stress test. This is indeed a good project.

**Comment by anon238** [comment]
I know that Arc doesn’t use a lot recourses and are mostly very cheap to host like 5-10 bucks a month but honestly I don’t know anything about that programming language and there is also not a tutorial for it. I don’t want to scale each month to a higher plan since I won’t make any money out of it. I’m looking for a platform that uses less recourses but I don’t know which one, there are too many to choose from.
